src/k8s/requests/v2v/constants.js

```diff
@@ -7,3 +7,10 @@ export const CONVERSION_GENERATE_NAME = `${CONVERSION_BASE_NAME}-`;
 export const CONVERSION_PROGRESS_ANNOTATION = 'v2vConversionProgress';
 
 export const V2VVMWARE_DEPLOYMENT_NAME = 'v2v-vmware';
+
+export const VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACE = 'openshift'; // along the common-templates ...
```

Comment:
is the openshift namespace really the right one we want?

Reply:
Any better idea? :-)

Reply:
Must be accessible but still "safe"

- kubevirt is gone
- kube-public is dangerous
- kube-system is simply not right
- openshift already contains similar "common-templates" stuff

I do not want to introduce new one ...

Reply:
why is the `kube-public` dangerous?

I guess `openshift` will do, we do not want to introduce another namespace

Reply:
I consider `kube-public` more like `/tmp` on unix.
I think there's higher probability that an "untrusted" user can get privileges to modify ConfigMaps in `kube-public` and so (s)he would be able to replace image (repo + name + tag) which is used for execution by another user.

Seems like `openshift` is the best option here so far.

Reply:
ok